checkbox-provider-ce-oem/units/touchscreen/jobs.pxu

```diff
@@ -0,0 +1,48 @@
+unit: template
+template-resource: com.canonical.certification::device
+template-filter: device.category == 'TOUCHSCREEN'
+template-engine: jinja2
+template-unit: job
+plugin: user-interact
+category_id: com.canonical.plainbox::touchscreen
+id: ce-oem-touchscreen/evdev/single-touch-tap-{{ product_slug }}
+imports: from com.canonical.certification import device
+estimated_duration: 10.0
+_description:
+  PURPOSE:
```

Comment:
What do you think if we write it like `_purpose`, `_steps` or  `_verification`? Cause this kind of style is supported in checkbox.